app/views/components/docs/contents-list.yml

```diff
@@ -38,6 +39,17 @@ examples:
           text: Another pretty long contents list entry, not as long as the first, but still a little.
         - href: "#third-thing"
           text: Third thing
+  active content link:
+    description: 'An active state allows for "you are here" items in a contents list that spans different pages to avoid linking back to the current page.'
+    data:
+      contents:
+        - href: "#first-thing"
+          text: First
+        - href: "#two"
+          text: Second
```

Comment:
Using Voiceover, I've noticed the active text is read out differently to the actual links when it comes to separating out the dash before each list item. The actual links separate out the two: "emdash" "First". Whereas the active text reads like: 'comma Second'

http://government-frontend.dev.gov.uk/component-guide/contents-list/active%20content%20link/preview

Reply:
Have replicated locally, but I'm reluctant to change anything as it would seem to be a case of trying to "fix" things specifically for the screenreader, which is generally advised against. In this case it would be awkward to add a span because of the use of link_to_if

Reply:
@maxgds Having looked at the original PR for the contents list component, it looks like reading out the emdash perhaps wasn't ideal behaviour anyway, so makes sense to leave it as is for that reason too. 👍